alacritty/src/migrate/mod.rs

```diff
@@ -327,6 +376,79 @@ root_value = 3
         assert_eq!(output, expected);
     }
 
+    fn mouse_hints_template(input: &str, expected: &str) {
+        let mut document = input.parse::<DocumentMut>().unwrap();
+
+        let _ = migrate_mouse_hints(&mut document);
+
+        let output = document.to_string();
+
+        assert_eq!(output, expected);
+    }
+
+    #[test]
+    fn mouse_hints() {
+        // true
+        mouse_hints_template(
+            r#"
+[other_stuff]
+foo = 5
+bar = 7
+
+[[hints.enabled]]
+mouse.enabled = true
+        "#,
+            r#"
+[other_stuff]
+foo = 5
+bar = 7
+
+[[hints.enabled]]
+mouse.button = "Left"
+        "#,
+        );
```

Comment:
If you follow the existing tests and make the format string into a variable, you'll avoid all these extra lines due to odd automatic formatting.